src/containers/intake/schemas/IntakeSchemas.js

```diff
@@ -236,6 +243,12 @@ const dataSchema = {
 
 const uiSchema = {
   [getPageSectionKey(1, 1)]: {
+    classNames: 'column-span-12 grid-container',
+    [getEntityAddressKey(0, FORM, DATETIME_ADMINISTERED)]: {
+      classNames: 'column-span-4'
+    },
+  },
```

Comment:
Widget? 😀

Reply:
ooh you only have to put `format: 'date'` in the schema for the property to be a DateWidget!

Reply:
good to know! Thanks :)